src/components/creator/Creator.jsx

```diff
@@ -23,7 +23,7 @@ export default function Creator(props) {
             onDelete={onDelete}
             options={options}
             hidden={!!item._destroy} // eslint-disable-line no-underscore-dangle
-            selected={item[selectedField]}
+            selected={item[selectedField].toString()}
```

Comment:
I would argue instead for the removal of the `selectedField` prop. We know that it's only set in one place (see the snippet bellow). The components themselves seem to offer little value in reusability (it's only a list of pairs of inputs (one select, and one text). I also don't know how to systemically handle the absence of a `bme_id`: I can make this part of the form fail gracefully, but I still don't know how the form as a whole should behave if the field is not there (which probably indicate a schema misalignment with the API).

```
$ git grep selectedField
src/components/creator/Creator.jsx:6:  const { title, items, onAdd, onEdit, onDelete, options, selectedField } = props;
src/components/creator/Creator.jsx:19:            key={item[selectedField]}
src/components/creator/Creator.jsx:26:            selected={item[selectedField].toString()}
src/components/creator/Creator.jsx:28:            selectedField={selectedField}
src/components/creator/Creator.jsx:33:      <CreatorItem options={options} onAdd={(...args) => onAdd(...args)} selectedField={selectedField} />
src/components/creator/Creator.jsx:45:  selectedField: PropTypes.string
src/components/creator/Creator.jsx:49:  selectedField: 'id'
src/components/creator/CreatorItem.jsx:23:      selectedField
src/components/creator/CreatorItem.jsx:45:              onChange={item => action({ [selectedField]: item.value, description: this.input.value }, index)}
src/components/creator/CreatorItem.jsx:70:  selectedField: PropTypes.string,
src/components/pages/StudyCase/EditStudyCasePage.jsx:414:            selectedField="bme_id"
$ git grep CreatorItem
src/components/creator/Creator.jsx:2:import CreatorItem from 'components/creator/CreatorItem';
src/components/creator/Creator.jsx:15:      {/* One CreatorItem per item filled with its data */}
src/components/creator/Creator.jsx:18:          <CreatorItem
src/components/creator/Creator.jsx:32:      {/* CreatorItem for adding new items */}
src/components/creator/Creator.jsx:33:      <CreatorItem options={options} onAdd={(...args) => onAdd(...args)} selectedField={selectedField} />
src/components/creator/CreatorItem.jsx:6:export default class CreatorItem extends React.Component {
src/components/creator/CreatorItem.jsx:64:CreatorItem.propTypes = {
src/components/creator/CreatorItem.jsx:77:CreatorItem.defaultProps = {
```